src/mayo.c

```diff
@@ -599,51 +442,28 @@ int mayo_expand_sk(const mayo_params_t *p, const unsigned char *csk,
     }
 #endif
 
-    // compute P_i^(1) + P_i^(1)t for all i
-    int used = 0;
-    for (int r = 0; r < param_v; r++) {
-        for (int c = r; c < param_v; c++) {
-            if (r == c) {
-                memset((void *)(bitsliced_P1_P1t + m_legs * 4 * (r * param_v + c)), 0,
-                       param_m / 2);
-            } else {
-                bitsliced_m_vec_copy(m_legs, bitsliced_P1 + m_legs * 4 * used,
-                                     bitsliced_P1_P1t + m_legs * 4 * (r * param_v + c));
-                bitsliced_m_vec_copy(m_legs, bitsliced_P1 + m_legs * 4 * used,
-                                     bitsliced_P1_P1t + m_legs * 4 * (c * param_v + r));
-            }
-            used++;
-        }
-    }
-
     // compute L_i = (P1 + P1^t)*O + P2
     uint32_t *bitsliced_L = bitsliced_P2;
 
-    P1P1t_times_O(p, bitsliced_P1_P1t, O, bitsliced_L);
```

Comment:
Since `bitsliced_L` is passed via the public API, the AVX2 version of P1P1t_times_O needs to be updated to accept unaligned memory.